app/controllers/api/v1/cards_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Api::V1::CardsController < ApplicationController
   def index
-    render json: Card.sort_deck
+    render json: UserCard.sort_deck
```

Comment:
@justinetroyke  Good call changing this to a UserCard method.   Eventually, we will need to decode a JWT to obtain a user id and pass it to this method, so we are retrieving one user_cards associated with the current user.